### ISSUE TITLE
Selecting a grabbable with a ray interactor randomly snaps it closer

### ISSUE THREAD

**Issue by endasil** [Unity-Technologies/XR-Interaction-Toolkit-Examples#68]
I select a grabbable with a ray interactor and have anchor control active. Expected result: I gain control of the object where it is. The actual result: the object moves closer every time I grab it and eventually move into the player. 

Is there a workaround for this bug?

**Comment by GSOsborne** [comment]
Yeah, check the "Use Dynamic Attach" box so that the intersection of your ray with the object becomes the attach point.

![image](https://github.com/Unity-Technologies/XR-Interaction-Toolkit-Examples/assets/44345509/3a27c993-b724-4f63-bfca-57cd42f4d424)
